libs/datasets/timeseries.py

```diff
@@ -782,6 +782,39 @@ def add_new_cases(timeseries: MultiRegionTimeseriesDataset) -> MultiRegionTimese
     return new_timeseries
 
 
+def _calculate_modified_zscore(series: pd.Series, window: int = 10) -> pd.Series:
+    """Calculates zscore removing zeros"""
+    series = series.copy()
+    series[series == 0] = None
+    rolling_series = series.rolling(window=window, min_periods=3)
```

Comment:
Make the default explicit either here or in the docstring that this is a trailing, right inclusive equally weighted window.